### ISSUE TITLE
Wrapping an element around WorldView component causes infinite increase in height

### ISSUE THREAD

**Issue by spjy** [cruise-automation/webviz#205]
Whenever I wrap an HTML element around the WorldView component, it infinitely increases its height. I am forced to set a fixed height and width as a result.

See https://codesandbox.io/s/relaxed-sun-tqxk3.

**Comment by vidaaudrey** [comment]
Hi @spjy  that's a known issue. For now, you could try applying `height: "100%" ` to the Worldview wrapper div to make sure by default Worldview fills up the whole area. 
https://codesandbox.io/s/quizzical-kalam-ducme

**Comment by spjy** [comment]
Right, that is what I am doing now; however that is a band aid solution and is also impractical in my case as I have a wrapper component that requires block level elements.

**Comment by jtbandes** [comment]
Can you describe more about how you expect the Worldview component should be sized in your case? Is something constraining its size from the parent down, or how do you expect it to choose its size?

(FYI the issue is a side effect of how we're using [react-container-dimensions](https://www.npmjs.com/package/react-container-dimensions) inside Worldview)

**Comment by spjy** [comment]
I expect it to be sized based on its immediate parent container.
